ntoskrnl/mm/ARM3/procsup.c

```diff
@@ -889,6 +894,7 @@ MiInsertSharedUserPageVad(VOID)
         ExFreePoolWithTag(Vad, 'ldaV');
         return Status;
     }
+    PsChargeProcessNonPagedPoolQuota(Process, sizeof(MMVAD_LONG));
```

Comment:
This will lead to a compilation failure as `Process` is an unintialized and undefined variable. You could adapt `MiInsertSharedUserPageVad` by taking an input parameter argument to a `EPROCESS` since the function is called by `MmInitializeProcessAddressSpace` and it provides a process object.